helper_scripts/create_gpg_keys.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+import argparse
+import gnupg
+import os
+import shutil
+import scriptworker.gpg
+
+
+class FullPaths(argparse.Action):
+    """Expand user- and relative-paths"""
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, os.path.abspath(os.path.expanduser(values)))
+
+parser = argparse.ArgumentParser(description="GPG keypair creation and export script")
+parser.add_argument('-b', '--gpgbin', nargs='?', help="GPG binary used", action=FullPaths, default="/usr/local/bin/gpg2")
```

Comment:
What I wanted to know regarding the defaults for the arguments is if we should use the environment variables set by `gpg_helper.sh` instead of the hardcoded defaults already in place.